### ISSUE TITLE
Full Name does not appear as a field in AOR Reports

### ISSUE THREAD

**Issue by jason-huett** [salesagility/SuiteCRM#1430]
Sugar Pro - layouts and reports allow selection of "Full Name" for users, contacts, employees.  SuiteCRM shows Full Name for layouts, but not for AOR Reports.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Full Name does not appear as a field in AOR Reports](https://trello.com/c/UNzo5mCo/96-full-name-does-not-appear-as-a-field-in-aor-reports)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
